data/operators/amenity/police.json

```diff
@@ -2097,24 +2085,15 @@
         "operator": "Polizei Viersen"
       }
     },
-    {
-      "displayName": "Polizia",
-      "id": "polizia-f71f16",
-      "locationSet": {"include": ["it"]},
-      "tags": {
-        "amenity": "police",
-        "operator": "Polizia"
-      }
-    },
     {
       "displayName": "Polizia di Stato",
       "id": "poliziadistato-f71f16",
       "locationSet": {"include": ["it"]},
+      "matchNames": ["polizia"],
```

Comment:
you let this ```polizia di stato``` to define a general ```polizia```.

I guess the word polizia means police station in your language? If this word is usually define this police station network, this matchname is OK

Reply:
If you think this matchName is correct, just have not to do anything about what I just said.

Reply:
Yes, exactly, Polizia is just the common name we use for Polizia di Stato, no one calls it Polizia di Stato, we say Polizia for Polizia di Stato and Carabinieri for Arma dei Carabinieri as we have two "police", one is administrated by the Ministry of the Interior (Polizia di Stato) and one administrated by Ministry of Defence (Carabinieri, military police), so they are different and so is the term to define them. I hope to have explained it well, I'm not very good at explaining things 😅

Reply:
> 
> 
> Yes, exactly, Polizia is just the common name we use for Polizia di Stato, no one calls it Polizia di Stato, we say Polizia for Polizia di Stato and Carabinieri for Arma dei Carabinieri as we have two "police", one is administrated by the Ministry of the Interior (Polizia di Stato) and one administrated by Ministry of Defence (Carabinieri, military police), so they are different and so is the term to define them. I hope to have explained it well, I'm not very good at explaining things 😅

Thanks! You are very patient to explain this thing. So this pieces of review suggestion is my none-understanding of italy. 👍👍

Reply:
Don't worry, Italy is a bit complicated itself, the result of many years of history and various cultures 😂